snorkel/map/core.py

```diff
@@ -74,6 +68,7 @@ def run(self, **kwargs: Any) -> FieldMap:
         raise NotImplementedError
 
     def __call__(self, x: DataPoint) -> DataPoint:
+        x = dill.loads(dill.dumps(x))
```

Comment:
Huh?

Reply:
At least add insider-focused comments for this sort of cryptic thing...

Reply:
Will do. Just a more reliable way to deepcopy (e.g. calling deepcopy on a pd.Series or SimpleNamespace with a dictionary attribute won't dereference the dictionary)